functional-tests/src/test/resources/GebConfig.groovy

```diff
@@ -15,7 +15,7 @@ import org.openqa.selenium.safari.SafariDriver
 import org.openqa.selenium.remote.DesiredCapabilities
 
 waiting {
-  timeout = 20
+  timeout = 60

```

Comment:
We should be a bit careful of increasing this too high.  

Do we really expect a single operation to take up to 60 seconds in OpenShift?
This timer isn't for the entire test to run, just each individual operation or assertion.

Though to be fair, in happy path cases, the tests will run as fast as possible and this timeout will never be hit.

In cases where tests fail, this will make the overall test run slower since it will wait for 1 minute to timeout for each failing test.

Only matters if the speed of the pipeline in failing cases matters.